lesson6.html

```diff
@@ -62,53 +76,48 @@ <h3>Dictionaries</h3>
         <section>
           <h3>Dictionaries</h3>
           <p>
-            The indices are called <b>keys</b>.<br>
-            Each key has a <b>value</b> associated with it.<br>
-            So the elements are called <b>key-value pairs</b>.<br><br>
-
+            You can also create a dictionary with data in it:
             <div class="fragment">
```

Comment:
If this is here, the code will show up after you click next:
![image](https://user-images.githubusercontent.com/11900207/62286295-54d25f80-b44f-11e9-9708-22381967ebe5.png)
and then
![image](https://user-images.githubusercontent.com/11900207/62286309-5b60d700-b44f-11e9-91c4-0c376a17ae91.png)

And the same for line 100

Reply:
nice catch Ana, thanks! and added fragments for the questions :)